docs/_jun_milestone_.md

```diff
@@ -0,0 +1,25 @@
+# Monthly Milestone - Jun 2020
+
+## Overview
+
+- Finish Q2 release
+
+## Goals
+
+- [ ] Clean up extension template and docs #5336
+- [ ] Failed to exclude properties by setting false in `fields` filter via url
+      #4992
+- [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+- [ ] Integration with TypeORM #4853
+
+## Stretch Goals
+
+- [ ] Documentation improvement
+
+  - [ ] [Docs] MongoDB connector docs should cover LB4 content #4879
+  - [ ] [Docs] MySQL connector docs should cover LB4 content #4878
+  - [ ] [Docs] Oracle connector docs should cover LB4 content #4880
+  - [ ] [Docs] Add "project layout reference" page #4847
+
+- [ ] [Workshop] From API to UI: LoopBack with Angular with client generator
+      #4849
```

Comment:
(Proposal) PRs to review:
- [ ] [RFC] Enable parallel mocha testing #5011 (IIUC, waiting for the next mocha release)
- [ ] feat(http-server): adds http/2 support #4989 (probably need more review time?)
- [ ] feat(graphql): add an extension for type-graphql integration #5545

Reply:
@dhmlau 
> feat(http-server): adds http/2 support #4989 (probably need more review time?)

This PR is waiting for Express's support of http2, please see discussion https://github.com/strongloop/loopback-next/pull/4989#issuecomment-625380889 and https://github.com/strongloop/loopback-next/pull/4989#issuecomment-633346475.

> RFC] Enable parallel mocha testing #5011 (IIUC, waiting for the next mocha release)

Yep, it's waiting for mocha release.